pkg/pb/node.proto

```diff
@@ -46,7 +46,7 @@ message NodeAddress {
 enum NodeTransport {
     TCP_TLS_GRPC = 0;
 }
-// NodeStats is info about a single storagenode stored in the stats db
+// NodeStats is info about a single storagenode
```

Comment:
I think "NodeStats is the reputation characteristics of a node" or something like that would make more sense for documenting this